app/code/Magento/GiftMessage/view/frontend/web/js/model/gift-message.js

```diff
@@ -126,9 +131,7 @@ define([
              * After submit.
```

Comment:
Please, leave comment in JSDoc  that method is empty, but it`s present to provide extension point.